src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs

```diff
@@ -130,6 +131,16 @@ public bool PortDefaultValueMarkerVisible
             }
         }
 
+        public string ValueMarkerWidth
```

Comment:
/// summary tags for public properties please.

Reply:
property removed